Microsoft.Toolkit.Uwp.UI/Extensions/ScrollViewer/ScrollViewerExtensions.cs

```diff
@@ -11,7 +10,7 @@
 namespace Microsoft.Toolkit.Uwp.UI.Extensions
 {
     /// <summary>
-    /// Provides attached dependency properties for the <see cref="Windows.UI.Xaml.Controls.ListViewBase"/>
+    /// Provides attached dependency properties for the <see cref="ListViewBase"/>
```

Comment:
This should be ScrollViewer, no? :P

Reply:
Whoops! 😅

Fixed in https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3220/commits/31a1b8dd3e554e2686fa1716244389dc618c3a43.

Reply:
You fixed it only on the ScrollViewerExtensions.ExpressionAnimations.cs class, not on the ScrollViewerExtensions.cs
Would you mind fixing it here as well?

Reply:
Ah, so that's where that typo was originally coming front! 😄

Fixed in https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3220/commits/5c366a473a514c4679c41999f5471d8849a271ac for all `ScrollViewer` extension classes.